test/kdump_test.rb

```diff
@@ -159,4 +167,290 @@
       end
     end
   end
+
+  describe ".ReadKdumpKernelParam" do
+    before do
+      allow(Yast::Bootloader).to receive(:kernel_param).and_return kernel_param
+      Yast::Kdump.ReadKdumpKernelParam
+    end
+
+    let(:is_crashkernel_set) { Yast::Kdump.crashkernel_param }
+    let(:crashkernel_needed) { Yast::Kdump.add_crashkernel_param }
+    let(:several_ranges) { Yast::Kdump.crashkernel_list_ranges }
```

Comment:
Both variables are confusing, it's a boolean variable, but that's not understandable from their name.